package.json

```diff
@@ -81,6 +81,7 @@
     "prop-types": "^15.7.2",
     "react": "~16.8.1",
     "react-dom": "~16.8.1",
+    "react-pdf-js": "^5.1.0",
```

Comment:
this should be a `peerDependancy` read more about it (here)[https://nodejs.org/en/blog/npm/peer-dependencies/#using-peer-dependencies]